Resources/Storage/DIPStorage.py

```diff
@@ -27,6 +27,10 @@
 
 class DIPStorage( StorageBase ):
 
+  _InputProtocols = ['file', 'dip', 'dips']
+  _OutputProtocols = ['dip', 'dips']
```

Comment:
If these are immutable constant lists, then names must be in all capitals, e.g. INPUT_PROTOCOLS